### PR TITLE
Rewrite for better readability

### PR DESCRIPTION
**Pull request by eraykoschell**

### CODE REVIEW

**Comment by eraykoschell** [comment]
Also, allowed bot to be used in bot-construction channel _(forgot to mention it in commit)_.

File: bot.py [hunk 1]
```diff
@@ -9,24 +9,31 @@ async def on_ready():
...
+  if cmd == "test()":
+    await msg.channel.send("Test worked you didnt break it yet")
 
-  if "crash()" == message.content.lower() and 'commands' == str(message.channel):
-    await message.channel.send("Nice try I am impossible to crash")
+  if cmd == "schedule()":
+    await msg.channel.send("Melkey streams start on Mondays, Wednesdays, and Fridays at 9PM EST")
+
+  if cmd == "project()":
+    await msg.channel.send("Melkey is working on a NBA app written in react to search, and compare player stats! You can find it over at https://github.com/Amokstakov/NewNBAApp")
```
Comment (ghost) [inline bot.py]: we need to update this due to the repo going private

**Review by ghost — APPROVED** [review]
looks good

File: bot.py (hunk 1 above)
Comment (eraykoschell) [reply bot.py]: I'll remove the link for now and then in the future, replace it with the link to the production website instead.

**Pull request merged**